src/shared/sol-random.c

```diff
@@ -86,7 +86,9 @@ getrandom_shim(void *buf, size_t buflen, unsigned int flags)
 static uint64_t
 get_platform_seed(uint64_t seed)
```

Comment:
By the same algorithm, you mean Mersenne Twister? If so, it is only as good as the initial seed; if it's initialized to `time(NULL)`, it's not a really good starting point.

It might be a good idea later, then, to ensure that the RIOT implementation of `SOL_RANDOM_MT19937` uses the system implementation.